css/abstract/colors.css

```diff
@@ -1,15 +1,16 @@
-:root{
-    --primary-color: #fa643f;
-    --border-color:#d5d5d5;
-    --btn-color:#fff0ec;
-    --slider-bg-color:#d3d3d3;
-    --white-color:#ffffff;
-    --black-color:#000000;
-    --card-badge-bg-color:#fa643fad;
-    --footer-bg-color:rgba(0, 0, 0, 0.863);
-    --header-bg-color:rgb(255, 255, 254);
-    --form-bg-color: #f5f3f3;
-    --form-btn-color:#741802;
-    --grey-color:grey;
-    --link-color:rgb(18, 166, 224);
-}
\ No newline at end of file
+:root {
+  --primary-color: #fa643f;
+  --primary-bg-color:rgb(209, 80, 41);
```

Comment:
Try to use one type of color scheme either hex or rgb

Reply:
You can convert online your rgb to hex. You can try that

Reply:
ok will make the changes. Thank you.